File: DSA/Array/C/reversearray.c
#include <stdio.h>

void printReverse(int arr[], int len){
    
    for(int i = len - 1; i >= 0; i--)
        printf("%d ", arr[i]);
}

int main()
{
    int arr[] = {10, 20, 30, 40, 50, 60};
    
    int len = sizeof(arr)/sizeof(arr[0]);
    
    printf("Array in Reverse:\n");
    printReverse(arr, len);

    return 0;
}

File: DSA/Array/Java/Reverse_Array.java
import java.util.*;


public class Reverse_Array {
    public static void main(String[] args) {
        int size;
        Scanner ob1 = new Scanner(System.in);
        
        System.out.println("Enter the size of the array : ");
        size = ob1.nextInt();
        int arr[] = new int[size];
        
        System.out.println("Enter the elements of the array : ");
        for (int i=0; i<size ; i++) {
            arr[i] = ob1.nextInt();
        }
        System.out.println("Now the array is : ");
        for (int i=0; i<size ; i++) {
            System.out.print(arr[i]+" ");
        }
        
        
        int revArr[] = new int[size];
        System.out.println("\nNow the reverse array is : ");
        for (int i =size-1; i >=0 ; i--) {
            System.out.print(arr[i]+" ");
            revArr[i] = arr[i];
        }
        System.out.println();
    }
    
    
    
}
